src/pages/admin/EventEdit.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
 import * as Yup from "yup"
 import { Formik } from "formik";
+import { setEvent } from "../../actions/PageActions";
 import EditEventForm from '../../components/Forms/EditEvent'
 import { fetchBackend } from '../../utils'
 import { connect } from "react-redux";
 import { Helmet } from 'react-helmet';
 
 function EventEdit(props) {
+
+    const { id: eventId } = useParams();
+
+    const { setEvent, event, events } = props;
+
+    useEffect(() => {
+        // Reupdate only if event does not exist, and there are events to filter
+        if(!event && events && eventId) {
+            setEvent(events.find(event => event.id === eventId));
```

Comment:
I know it was like this before, but do you think it's necessary to store the 'current' event in redux? Maybe we can just use state on the event pages

Reply:
I thought the same.. As of now, I can't really think of a reason where it would be useful to store individual events except maybe to store a `loaded` boolean state for the page itself.

Reply:
If you would like to remove it, feel free to